### PR TITLE
Feature/accounts

### PR DESCRIPTION
**Pull request by SPICYJO**
Feature : Accounts management
[Django backend]
- Add custom user model and authentication backend
- Add accounts related RESTful API
- Add accounts related unit test
- Add Sphinx automatic documentation generator
[React frontend]
- Add SignIn, SignUp pages
- Add AccountBox (which shows username and sign-out button or sign-in button)
- Install new packages (redux-action, redux-pender, material-ui, etc)
- [TODO] Unit test for accounts related features needs to be added (Since accounts related features could be a critical feature for other features and my branch is getting quite far from master branch (also I do not want other team members suffer installing packages again), I created this pull request despite lack of FE test case)

### CODE REVIEW

File: frontend/src/components/accounts/SignIn.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+      <Link color='inherit' href='/'>
+        Term&#39;inator
+      </Link>{' '}
+      {new Date().getFullYear()}
+      {'.'}
+    </Typography>
+  )
+}
+
+const useStyles = makeStyles((theme) => ({
```
Comment (qwerty98765) [inline frontend/src/components/accounts/SignIn.js:34]: style에 관한 부분은 따로 css파일로 빼는 것도 괜찮을 것 같아요!

File: frontend/package.json [hunk 2]
```diff
@@ -3,10 +3,16 @@
   "version": "0.1.0",
```
Comment (qwerty98765) [inline frontend/package.json:3]: package.json 같은 파일들은 gitignore에 넣어도 괜찮을 것 같습니다!

Comment (qwerty98765) [reply frontend/package.json:3]: 이 외에도 package-lock.json이나 migrations.__init__.py, yarn.lock과 같은 파일들도 그렇습니다!

File: frontend/src/components/accounts/SignIn.test.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+// import React from 'react';
```
Comment (qwerty98765) [inline frontend/src/components/accounts/SignIn.test.js:1]: 주석은 필요한 부분이 아니라면 빼주셔도 괜찮을 것 같아요 :)

Comment (qwerty98765) [reply frontend/src/components/accounts/SignIn.test.js:1]: 필요한 주석이면 어떻게 쓸 예정인지 달아주세요. 반영해주실 것이라면 다른 파일들도 그렇게 해주시면 더 좋을 것 같아요 👍

File: backend/docs/index.rst [hunk 4]
```diff
@@ -0,0 +1,24 @@
+.. Term'inator documentation master file, created by
```
Comment (qwerty98765) [inline backend/docs/index.rst:1]: .rst 파일들과 .bat 파일들을 어떻게 작성하는 것인지 잘 몰라서 혹시 조금 설명해주실 수 있을까요 ♿

**Review by qwerty98765 — APPROVED** [review]
전체적으로 너무 고생하셨네요 😆 .rst나 .bat등은 comment에도 남겨두었지만, 다음 미팅 때 설명 조금만 부탁드립니다. 일단 저는 approve할게요!

File: backend/docs/index.rst (hunk 4 above)
Comment (SPICYJO) [reply backend/docs/index.rst:1]: These files are automatically generated by Sphinx which is a documentation generator. We have no need to change these files unless we want to configure our documentation settings later.

File: frontend/src/components/accounts/SignIn.test.js (hunk 3 above)
Comment (SPICYJO) [reply frontend/src/components/accounts/SignIn.test.js:1]: I tried to create unit tests for FE components but unfortunately it has failed. So I just commented out whole codes and left only dummy test to success build on Travis. I will fix it soon.

File: frontend/package.json (hunk 2 above)
Comment (SPICYJO) [reply frontend/package.json:3]: Well, I am not sure what you want to say. I think package.json should be included(?!). [Link : https://stackoverflow.com/questions/46174085/should-changes-in-a-package-json-file-be-commited-to-a-repository-as-well]

**Comment by SPICYJO** [comment]
Fix Travis failure

File: frontend/package.json (hunk 2 above)
Comment (qwerty98765) [reply frontend/package.json:3]: Oh, I see all these files look like they should be included. Although there seems to be some controversy about whether to include migration files in the commit(https://stackoverflow.com/questions/28035119/should-i-be-adding-the-django-migration-files-in-the-gitignore-file), it seems fine to commit them.

**Comment by qwerty98765** [comment]
> Fix Travis failure

Lets merge as soon as another member checks this PR. 😄

**Pull request merged**